### PR TITLE
Add hamburger menu to topbar component

### PR DESCRIPTION
**Pull request by atorczynski**
Menu with Y-Axis transition from top.

### CODE REVIEW

File: client/src/components/BurgerButton.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    width: 2rem;
+    height: 0.25rem;
+    background: black;
+    border-radius: 10px;
+    transition: all 0.3s linear;
+    position: relative;
+    transform-origin: 1px;
+
+    :first-of-type {
+      transform: ${props => (props.open ? 'rotate(45deg)' : 'rotate(0)')};
```
Comment (lmachens) [inline client/src/components/BurgerButton.js]: Isn't there an easier way to set css if the component is open?
`rotate(0)`, `opacity: 1` ,.. doesn't look required.

Does something like this work?

```
     ${props =>props.open ? `
      :first-of-type {
        transform: rotate(45deg);
      }

      :nth-of-type(2) {
      opacity: 1;
      transform: translateX(20px);
    }
     `: ''};
```

Or make use of https://emotion.sh/docs/composition

```
const open = css`
      :first-of-type {
        transform: rotate(45deg);
      }

      :nth-of-type(2) {
      opacity: 1;
      transform: translateX(20px);
    }
`
```

```
  ${props => props.open ? open : ''}
```

File: client/src/components/Logo.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import Logosvg from '../assets/logo.svg';
 import styled from '@emotion/styled';
+import { Link } from 'react-router-dom';
 
 const LogoDisplay = styled.img`
-  height: 200px;
-  width: 100%;
+  height: 125px;
+  margin-right: -40px;
```
Comment (lmachens) [inline client/src/components/Logo.js]: `margin-right` shouldn't be part of the logo. Instead, it's up to the parent to position the logo.

File: client/src/components/Menu.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    text-decoration: none;
+    transition: color 0.3s linear;
+  }
+`;
+
+export default function Menu(props) {
+  return (
+    <StyledMenu open={props.open}>
+      <a href="/home">
+        <span role="img" aria-label="about us"></span>
```
Comment (lmachens) [inline client/src/components/Menu.js]:
```suggestion
        <span role="img" aria-label="about us" />
```
Happens multiple times

File: client/src/components/BurgerButton.js (hunk 1 above)
Comment (atorczynski) [reply client/src/components/BurgerButton.js]: Changed this one to emotion composition now. Works fine. 
`opacity: 1` changed to `opacity: 0` so middle `<div />` fades away on click.

File: client/src/components/Menu.js (hunk 3 above)
Comment (atorczynski) [reply client/src/components/Menu.js]: Thx for mentioning this one! Fixed it right away. Looks cleaner now.

File: client/src/components/Logo.js (hunk 2 above)
Comment (atorczynski) [reply client/src/components/Logo.js]: Deleted Logo component since it is only used within the appbar. Positioning takes place in parent component now.

**Comment by atorczynski** [comment]
this closes #65

**Pull request merged**